neue-kriminalpolitik.csl

```diff
@@ -0,0 +1,176 @@
+                <text variable="number"/>
+              </group>
+            </else>
+          </choose>
+          <choose>
+            <if match="any" variable="title-short">
+              <text variable="title-short" prefix=" &#8211; "/>
+            </if>
+          </choose>
+        </else-if>
+        <else>
+          <group delimiter=" ">
+            <text macro="authors-inline"/>
+            <date date-parts="year" form="text" variable="issued"/>
+          </group>
+          <choose>
+            <if match="any" variable="locator">
+              <text variable="page-first" prefix=", "/>
+              <text variable="locator" prefix=", "/>
+            </if>
+          </choose>
+        </else>
+      </choose>
+    </layout>
+  </citation>
+  <bibliography>
+    <sort>
+      <key macro="authors-bib"/>
+      <key variable="issued"/>
+    </sort>
+    <layout>
+      <choose>
+        <if type="article-journal" match="any">
+          <group delimiter=" ">
+            <text macro="authors-bib"/>
+            <text variable="title" suffix=","/>
+            <text term="in" suffix=":"/>
+            <text variable="container-title" form="short"/>
+            <text macro="number-and-year" suffix=","/>
+            <text variable="page"/>
+          </group>
+        </if>
+        <else-if type="book" match="any">
+          <group delimiter=" ">
+            <text macro="authors-bib"/>
+            <date date-parts="year" form="text" variable="issued" prefix="(" suffix=")"/>
+            <text variable="title"/>
+          </group>
+          <choose>
+            <if match="any" is-numeric="volume">
+              <text term="volume" prefix=", " suffix=" "/>
+              <number text-case="uppercase" variable="volume" form="roman"/>
+            </if>
+            <else-if match="any" variable="volume">
+              <text variable="volume" prefix=", "/>
+            </else-if>
+          </choose>
+          <choose>
+            <if match="any" is-numeric="edition">
+              <text variable="edition" prefix=", " suffix=". "/>
+              <text term="edition" form="short"/>
+            </if>
+          </choose>
+        </else-if>
+        <else-if type="legal_case" match="none">
+          <group delimiter=" ">
+            <text macro="authors-bib"/>
+            <date date-parts="year" form="text" variable="issued" prefix="(" suffix=")"/>
+            <text variable="title"/>
+          </group>
+          <choose>
+            <if match="all" variable="container-title">
+              <group delimiter=" " prefix=". ">
+                <text term="in" suffix=":"/>
+                <choose>
+                  <if match="any" variable="editor">
+                    <names variable="editor">
+                      <name form="short" delimiter=" / " et-al-min="2" et-al-use-first="1"/>
+                    </names>
+                    <text term="editor" form="short" prefix="(" suffix="),"/>
```

Comment:
you really want to do this with label, not with term -- see example above.

Reply:
Label does not have "editor", at least not in the online editor. The validator is unhappy, as well. I reverted the change, please let me know if i misunderstood something. Thanks!

Reply:
See the code I gave you for l. 33 above. Label doesn't take a creator type argument because it inherits that from names